internal/resources/metal/organization_member/models.go

```diff
@@ -0,0 +1,86 @@
+package organizationmember
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/packethost/packngo"
+)
+
+type ResourceModel struct {
+	ID             types.String `tfsdk:"id"`
+	Invitee        types.String `tfsdk:"invitee"`
+	InvitedBy      types.String `tfsdk:"invited_by"`
+	OrganizationID types.String `tfsdk:"organization_id"`
+	ProjectsIDs    types.Set    `tfsdk:"projects_ids"`
+	Nonce          types.String `tfsdk:"nonce"`
+	Message        types.String `tfsdk:"message"`
+	Created        types.String `tfsdk:"created"`
+	Updated        types.String `tfsdk:"updated"`
+	Roles          types.Set    `tfsdk:"roles"`
+	State          types.String `tfsdk:"state"`
+}
+
+func (m *ResourceModel) parse(ctx context.Context, invitee string, members []packngo.Member, invitations []packngo.Invitation) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	member, err := FindMember(invitee, members, invitations)
+	if err != nil {
+		m.OrganizationID = basetypes.NewStringNull()
```

Comment:
This `FindMember` call probably needs to added to each of the CRUD functions instead of keeping it here.  For example, I think the old SDKv2 resource had places where a nonexistent member was OK (when deleting an org member, it's fine for the org member to already not exist).  The `parse` function should take a `member` object as input.